addons/interaction/functions/fnc_canPardon.sqf

```diff
@@ -21,3 +21,4 @@ params ["_unit", "_target"];
 alive _target
 && {rating _target < -2000}
 && {side group _unit == side group _target}
+&& !{GVAR(disableNegativeRating)}
```

Comment:
I think this is not necessary since L22 will never be `true` anyways due to minimum rating being `0` when this setting is active.